trappy/base.py

```diff
@@ -179,6 +179,12 @@ def append_data(self, time, comm, pid, cpu, line, data):
         self.line_array.append(line)
         self.data_array.append(data)
 
+    def conv_to_int(self, value):
```

Comment:
Ok. How about we name it `try_parse_int`? I think that should make everyone happy hopefully